core/application/bloodDonationWorkflow/BloodDonationService.ts

```diff
@@ -25,36 +33,94 @@ export class BloodDonationService {
       })
       return 'We have accepted your request, and we will let you know when we find a donor.'
     } catch (error) {
-      throw new BloodDonationOperationError(`Failed to submit blood donation request. Error: ${error}`, GENERIC_CODES.ERROR)
+      if (error instanceof ThrottlingError) {
+        throw error
+      }
+      throw new BloodDonationOperationError(
+        `Failed to submit blood donation request. Error: ${error}`,
+        GENERIC_CODES.ERROR
+      )
+    }
+  }
+
+  private async checkDailyRequestThrottling(seekerId: string, repository: Repository<DonationDTO, DonationFields>, model: BloodDonationModel): Promise<void> {
+    const maxPostRequestPerDay = 10
```

Comment:
stuff like this should belong to a const file with clear indication that we should not change it

Reply:
moved the throttling-related constants to a dedicated constants file. updated at 95f5fa14b2c245af99af80f51f849f441df5c1a9